auth/server/src/main/java/com/seagates3/controller/AccountController.java

```diff
@@ -148,6 +151,20 @@ public ServerResponse create() {
 
         account.setEmail(email);
 
+        if (accessKey != null) {
+            AccessKey existingAccessKey;
+            try {
+                existingAccessKey = accessKeyDAO.find(accessKey);
+            } catch (DataAccessException ex) {
+                LOGGER.error("Failed to find access key in ldap -" + ex);
```

Comment:
Is this error is about failing to read from LDAP? and not about access key is not found in LDAP

Reply:
This error is about failing to read from LDAP.